react/components/ProductImages/components/Video/index.js

```diff
@@ -1,40 +1,41 @@
-import React, { Component } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
 import { match, compose, isEmpty, complement } from 'ramda'
 
 import Vimeo from './Vimeo'
 import YouTube from './Youtube'
 
-import styles from '../../styles.css'
+import { useCssHandles } from 'vtex.css-handles'
 
 const isNotEmpty = complement(isEmpty)
 
 const isVimeo = compose(isNotEmpty, match(/vimeo/))
 const isYoutube = compose(isNotEmpty, match(/youtube|youtu.be/)) 
 
-class Video extends Component {
-  static getThumbUrl(url, thumbWidth) {
-    if (isVimeo(url)) {
-      return Vimeo.getThumbUrl(url, thumbWidth)
-    }
+const CSS_HANDLES = ['productVideo', 'videoContainer', 'video']
 
-    else if (isYoutube(url)) {
-      return YouTube.getThumbUrl(url, thumbWidth)
-    }
+export function getThumbUrl(url, thumbWidth) {
+  if (isVimeo(url)) {
+    return Vimeo.getThumbUrl(url, thumbWidth)
   }
 
-  render() {
-    const { url } = this.props
-
-    return (
-      <div className={styles.video}>
-        {isVimeo(url) && <Vimeo {...this.props} />}
-        {isYoutube(url) && <YouTube {...this.props} />}
-      </div>
-    )
+  else if (isYoutube(url)) {
+    return YouTube.getThumbUrl(url, thumbWidth)
   }
 }
 
+function Video(props) {
+  const { url } = props
+  const handles = useCssHandles(CSS_HANDLES)
+
+  return (
+    <div className={handles.productVideo}>
+      {isVimeo(url) && <Vimeo {...props} cssHandles={handles} />}
```

Comment:
The vimeo component isn't receiving those `iframeContainer` and `iframe` handles, right?